src/libmeasurement_kit/neubot/dash_impl.hpp

```diff
@@ -0,0 +1,345 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+#ifndef SRC_LIBMEASUREMENT_KIT_NEUBOT_DASH_IMPL_HPP
+#define SRC_LIBMEASUREMENT_KIT_NEUBOT_DASH_IMPL_HPP
+
```

Comment:
Note to self: here we should probably add a reference to the paper with the methodology.